operations/research-and-development/organization/channels.md

```diff
@@ -1,80 +1,46 @@
-
-Areas of expertise:
-
-* Groups
-* LDAP
-* Permissions
-* System Console
-* Compliance
-* Data Retention
-* Mobile sidebar/categories
-
-### Interstellar (feature team #2)
-
-Areas of expertise:
-
-* Apps
-* Webhooks
-* Slash Commands
-* Plugins
-
-### Apollo (feature team #3)
-
-Areas of expertise:
-
-* Posts
-* Threads
-* File Previews
-
-## Platform Teams
-
-The platform teams are responsible for building and maintaining the systems that the feature teams rely on.
-
-**Why are these teams in the Channels product vertical and not the Platform division?**
-
-Until recently Channels was the only product Mattermost had and was itself the platform. As a result of this a lot of the platform is tightly coupled to the Channels product. The server and web app parts of the platform are especially tightly coupled. The goal is to eventually move these teams, or some version of them, out of the Channels product vertical when that coupling is looser.
-
-The teams are:
-
-### Web Platform
-
-The role of the Web Platform team is to own the infrastructure behind the Web App and Desktop App while supporting other teams that are using that infrastructure.
-
-Within the Web App, that means tooling (e.g. build tools, testing frameworks, style checking), core systems (e.g. Redux, routing), and helping maintain the core features of the app. For the areas owned, the team is also responsible for educating other teams about those areas and helping them coordinate work that might require changes to those things.
-
-The team is also responsible for dictating the coding style and practices used throughout the Web App which can be used as a reference for other JS projects. As recommended practices change, the team will work with the community and other teams to implement those in the Web App whenever possible.
-
-The team is not responsible for experimenting with new technologies or dictating the project structure/tooling used for all other JS projects within Mattermost. Because feature teams are able to move much quicker when they're writing brand new code, they have more opportunities to try out new technologies and practices. When those experiments are successful, the Web Platform team will help integrate them back into the Web App and make them available for other Mattermost projects.
-
-Areas of expertise:
-
-* Web App performance
-* Redux store structure
-* Desktop
-* Routing
-* Frontend coding style and best practices
-
-### Server Platform
-
-The role of the Server Platform team is to own the backend multi-product architecture and server performance while supporting other teams that rely on the server.
-
-Areas of expertise:
-
-* Multi-product and server platform archiecture
-* Load testing and server performance
-* Database requirements
-* Clustering and high availability
-* Backend coding style and best practices
+The Channels team is responsible for the Channels product within Mattermost including the Web, Mobile, and Desktop experiences of the product.
```

Comment:
> including the Web, Mobile, and Desktop experiences of the product

Does this imply the areas owned by web/desktop platform and mobile platform teams?

Reply:
Can you describe more what you mean?  I was trying to communicate that they own the Channels product across all clients.

Reply:
I noticed we didn't list the same for others, e.g. Boards or Playbooks. Is is because mobile experience is Channels-specific?

Reply:
Yes - for now it is.  Board and Playbooks lists their specific plugins. We can add mobile once it is available for these products.